{
  "Header": {
    "Original": "CVE-2018-7753 (An issue was discovered in Bleach 2.1.x before 2.1.3. Attributes that  ...)",
    "Line": 356954,
    "ID": "CVE-2018-7753",
    "Description": "(An issue was discovered in Bleach 2.1.x before 2.1.3. Attributes that  ...)"
  },
  "Annotations": [
    {
      "Original": "- python-bleach 2.1.3-1 (bug #892252)",
      "Line": 356955,
      "Type": "package",
      "Package": "python-bleach",
      "Kind": "fixed",
      "Version": "2.1.3-1",
      "BugNo": 892252
    },
    {
      "Original": "[stretch] - python-bleach \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 356956,
      "Type": "package",
      "Release": "stretch",
      "Package": "python-bleach",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "[jessie] - python-bleach \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 356957,
      "Type": "package",
      "Release": "jessie",
      "Package": "python-bleach",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "NOTE: https://github.com/mozilla/bleach/pull/356",
      "Line": 356958,
      "Type": "NOTE",
      "Description": "https://github.com/mozilla/bleach/pull/356"
    },
    {
      "Original": "NOTE: https://github.com/mozilla/bleach/commit/c5df5789ec3471a31311f42c2d19fc2cf21b35ef",
      "Line": 356959,
      "Type": "NOTE",
      "Description": "https://github.com/mozilla/bleach/commit/c5df5789ec3471a31311f42c2d19fc2cf21b35ef"
    }
  ]
}